cmd/create/operatorroles/by_prefix.go

```diff
@@ -195,6 +195,10 @@ func handleOperatorRoleCreationByPrefix(r *rosa.Runtime, env string,
 			r.Reporter.Infof(fmt.Sprintf("To create a cluster with these roles, run the following command:\n"+
 				"\trosa create cluster --sts --oidc-config-id %s --operator-roles-prefix %s%s",
 				args.oidcConfigId, args.prefix, hostedCpOutputParam))
+			if args.installerRoleArn != "" {
```

Comment:
I think this is only required if using a path right?

Reply:
Fixed, thanks.